### PR TITLE
Add tooltipAccessor prop to calendar to disable tooltip if needed

### LINKED ISSUE #226: Event title tooltip should stringify
It would appear that in case where we want to stylize (bold, italic, etc) some parts of an event title, it shows up properly on the page but the tooltip contains `[object Object]` as values.

A solution would be to use the `jsx-to-string` library to stringify the value or better yet add a tooltipAccessor method.

### PR DESCRIPTION
**Pull request by ginifizz**
Use a "tooltipAccessor" prop for the tooltip instead of the "titleAccessor" prop. Can show a tooltip different from the title, or not display it at all if the accessor returns null.

Fixes #226, #470

### CODE REVIEW

**Comment by wlalele** [comment]
:+1:

File: src/DayColumn.js [hunk 1]
```diff
@@ -230,7 +233,12 @@ class DayColumn extends React.Component {
               [isRtl ? 'right' : 'left']: `${Math.max(0, xOffset)}%`,
               width: `${width}%`,
             }}
-            title={(typeof label === 'string' ? label + ': ' : '') + title}
+            {...(tooltip
+              ? {
+                  title:
+                    (typeof label === 'string' ? label + ': ' : '') + tooltip,
+                }
+              : {})}
```
Comment (jquense) [inline src/DayColumn.js]: lets make this a bit less ugly please :P i'd just set title to `undefined` in the one case that will be fine

File: src/EventCell.js [hunk 2]
```diff
@@ -75,7 +78,7 @@ class EventCell extends React.Component {
           onClick={e => onSelect(event, e)}
           onDoubleClick={e => onDoubleClick(event, e)}
         >
-          <div className="rbc-event-content" title={title}>
+          <div className="rbc-event-content" {...tooltip ? { title: tooltip } : {}}>
```
Comment (jquense) [inline src/EventCell.js]: ditto here

**Comment by jquense** [comment]
thanks!

**Comment by RusselFourteen** [comment]
Can you please tell me how to off the built it Tooltip? I already tried many times but I still failed. Thanks and Sorry, newbie here

**Comment by RusselFourteen** [comment]
I already added toolTipAccessor but what is my next move? Where can I defined tooltip as undefined or null?

**Comment by ginifizz** [comment]
It works like the others accessors. Assign a function who returns NULL to the tooltipAccessor and it will be removed.
Like this : 
`<Calendar tooltipAccessor={() => null} ... />`

**Pull request merged**